### PR TITLE
Set queue address(es) to 'build' versions for testing

### PR DESCRIPTION
**Pull request by st3v3nhunt**
https://eaflood.atlassian.net/browse/PSD-956

Tested on https://github.com/DEFRA/ffc-demo-web/pull/185/. Successful [build](https://jenkins-ffc.azure.defra.cloud/job/ffc-demo/job/ffc-demo-web-build/job/psd-956-run-int-tests/10/) running all tests.

### CODE REVIEW

File: src/uk/gov/defra/ffc/Utils.groovy [hunk 1]
```diff
@@ -120,7 +120,7 @@ class Utils implements Serializable {
 
   static def getProvisionedQueueConfigValues(ctx, repoName, pr) {
     def configValues = [:]
-    def filePath = './provision.azure.yaml'
+    def filePath = Provision.azureProvisionConfigFile
```
Comment (markysoft) [inline src/uk/gov/defra/ffc/Utils.groovy:123]: it makes me nervous having Provision use Utils methods, and Utils use Provision methods as it introduces a cyclic dependency.

As these methods have Provision in the name, would they be better off in the Provision class?

**Review by markysoft — COMMENTED** [review]
Suggestion on relocation of Utils methods

File: src/uk/gov/defra/ffc/Utils.groovy (hunk 1 above)
Comment (st3v3nhunt) [reply src/uk/gov/defra/ffc/Utils.groovy]: Yup, good call.

**Review by markysoft — CHANGES_REQUESTED** [review]
Looks good, small suggestion on removing a variable, and a question about password characters.

File: src/uk/gov/defra/ffc/Provision.groovy [hunk 2]
```diff
@@ -177,6 +179,35 @@ class Provision implements Serializable {
...
+    def envVars = []
+
+    if(hasResourcesToProvision(ctx, azureProvisionConfigFile)) {
+      def queues = readManifest(ctx, azureProvisionConfigFile, 'queues')
+      queues.each {
+        envVars.push("${it.toUpperCase()}_QUEUE_ADDRESS=${getBuildQueuePrefix(ctx, repoName, pr)}$it")
+      }
+    }
+    envVars.push("MESSAGE_QUEUE_HOST=${appConfigValues[messageQueueHost]}")
+    envVars.push("MESSAGE_QUEUE_PASSWORD=${appConfigValues[messageQueuePassword]}")
```
[the remark this answers is not in the document]
Comment (st3v3nhunt) [reply src/uk/gov/defra/ffc/Provision.groovy:214]: I presume so, I'll make the change :+1:

**Review by markysoft — APPROVED** [review]

**Pull request merged**